heartpoints.sh

```diff
@@ -44,6 +59,70 @@ heartpoints_localDev() {
     error_and_exit "localDev is deprecated. For server side development, use ./hp serverDev or for client side development use ./hp clientDev"
 }
 
+string_toLower() { local stringToConvertToAllLowercase=$1
+    echo "$stringToConvertToAllLowercase" | tr '[:upper:]' '[:lower:]'
+}
+
+string_firstN() { local sourceString=$1; local lengthNOfDesiredCharsFromTheLeft=$2
```

Comment:
string_firstNChars may be a better name. This would also allow you to just use `N` as the 2nd parameter name!